Packs/ExtraHop/Integrations/ExtraHop_v2/ExtraHop_v2.py

```diff
@@ -1443,9 +1452,10 @@ def fetch_extrahop_detections(client: ExtraHopClient, advanced_filter: Dict, las
                 if detection_id not in already_fetched:
                     detection.update(get_mirroring())
                     incident = {
-                        'name': str(detection.get("type", "")),
+                        'name': str(detection.get("title", "")),
```

Comment:
Please explain this change.

Reply:
This is to better display the incidents in the XSOAR based on the more readable ExtraHop detection title rather then the ExtraHop type. 

Example: CVE-2019-0708 RDP Exploit Attempt > cve_2019_0708 (see below screenshots)

![2023-06-21 14_52_31](https://github.com/demisto/content/assets/61937657/6709edcd-c59c-4074-9d4f-30adb895966d)
![2023-06-21 14_52_10](https://github.com/demisto/content/assets/61937657/70044f12-f45a-4129-b4ce-f6ea5a1f8fb4)